### PR TITLE
Enable XML files to be selected in files dialogue

### LINKED ISSUE #49: Filetype check
Hi, at the moment using the the zip version there is no filetype check, if you select an image for example the importer returns 
"Success! Your import file is ready, let’s get started."

### PR DESCRIPTION
**Pull request by roborourke**
A feature of plupload is to set the mime types expected. This makes sure only xml files can be selected via the plupload popup.

Fixes #49

### CODE REVIEW

**Comment by roborourke** [comment]
@swissspidy hey can I be cheeky and ask for a review here? We're dog fooding the plugin on a client site for big imports and this has caused some issues so far.

Also if you know a better way to modify the media modal to remove the Upload tab let me know!

File: assets/intro.css [hunk 1]
```diff
@@ -8,3 +8,8 @@
 	width: 200px;
 	margin: 0 auto;
 }
+
+/* TODO: keep an eye out for a better way to remove the upload tab */
+#import-select .media-router a:not(.active) {
+	display: none;
+}
```
Comment (rmccue) [inline assets/intro.css]: This should be in a separate PR, not related to the XML selection as far as I can see.

Comment (roborourke) [inline assets/intro.css]: Yeah not entirely, just the popup modal. Think it needs more thought anyway

Comment (swissspidy) [inline assets/intro.css]: PR looks solid, but I agree that this part should be separated.

**Pull request merged**